machine.js

```diff
@@ -301,11 +301,15 @@ function Zeal8bitComputer() {
     }
 
     function restart() {
+        running = false;
```

Comment:
Good catch! This is required indeed.
It is also possible to clear the  `interval`, which is responsible for running the machine loop.
Maybe a more sturdy solution would be to call `stop()` which does all of that.

To prevent it from showing the CPU registers maybe we can add a boolean parameter?